index.html

```diff
@@ -192,5 +192,93 @@ <h2 class="card-title">profesional art printing date</h2>
+        <h1>About me</h1>
+        <p class="about-header-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="btn-about">
+          <span class="btn-about-text">get my resume</span>
+        </p>
+        <img
+          class="about-bg-img"
+          src="./images/about-bg.png"
+          alt="about background image"
+        />
+        <div class="about-content-divider">jjhg</div>
+      </div>
+      <div class="skills">
+        <div class="language-contaier">
+          <div class="diamond-language">
+            <span class="diamond"></span>
+            <span class="language">language</span>
+          </div>
+          <div class="lang">
+            <p class="lang-item java">
+              <span class="java-text">javascript</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">ruby</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">html</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">css</span>
+            </p>
+          </div>
+        </div>
+        <div class="language-contaier">
+          <div class="diamond-language">
+            <span class="square"></span>
+            <span class="language">Framework</span>
+          </div>
+          <div class="lang">
+            <p class="lang-item java">
+              <span class="java-text">bootstrap</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">ruby</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">RSPec</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">CapyBara</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">Selenium</span>
+            </p>
+          </div>
+        </div>
+        <div class="language-contaier">
+          <div class="diamond-language">
+            <span class="circle"></span>
+            <span class="language">skills</span>
+          </div>
+          <div class="lang">
+            <p class="lang-item java">
+              <span class="java-text">codekit</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">github</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">codepen</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">gitlab</span>
+            </p>
+            <p class="lang-item java">
+              <span class="java-text">terminal</span>
+            </p>
```

Comment:
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be in a `ul` list and each of them in a `li` tag 👍